streamer/src/sendmmsg.rs

```diff
@@ -135,14 +146,20 @@ where
     T: AsRef<[u8]>,
 {
     let size = packets.len();
-    #[allow(clippy::uninit_assumed_init)]
-    let iovec = std::mem::MaybeUninit::<iovec>::uninit();
-    let mut iovs = vec![unsafe { iovec.assume_init() }; size];
-    let mut addrs = vec![unsafe { std::mem::zeroed() }; size];
-    let mut hdrs = vec![unsafe { std::mem::zeroed() }; size];
+    let mut iovs = vec![MaybeUninit::<iovec>::uninit(); size];
+    let mut addrs = vec![MaybeUninit::<sockaddr_storage>::zeroed(); size];
```

Comment:
nit: there's no need to explicitly set the type, use MaybeUninit::uninit() and MaybeUninit::zeroed()

Reply:
Done in https://github.com/anza-xyz/agave/pull/3325/commits/67ff531cbb56c26c87e4197fe88ca883fa7e618a